src/System.Net.Sockets/System.Net.Sockets.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```

Comment:
I assume you'll remove the changes to this file before merging?

Reply:
Fixed